### PR TITLE
RWA 1305 added work type label

### PR DESCRIPTION
**Pull request by chrisdavecm**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-1305

### Change description ###
Added label

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaHelpers.java [hunk 1]
```diff
@@ -64,6 +64,7 @@ protected Task createMockedMappedTask() {
             new WarningValues(Collections.emptyList()),
             "someCaseManagementCategory",
             "hearing_work",
+            null,
```
Comment (serdaroguzhanakin) [inline src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaHelpers.java:67]: may add "Hearing work" instead of null

**Review by serdaroguzhanakin — COMMENTED** [review]
may add "Hearing work" instead of null

**Review by serdaroguzhanakin — DISMISSED** [review]

File: src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskBySearchForCompletablePactTest.java [hunk 2]
```diff
@@ -131,6 +131,7 @@ public List<Task> createTasks() {
             new WarningValues(Collections.emptyList()),
             "Case Management Category",
             "hearing_work",
+            "Hearing work",
```
Comment (BenNzewi) [inline src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskBySearchForCompletablePactTest.java:134]: could we also update the consumer (TaskManagementGetTasksBySearchForCompletableConsumerTest) to now include the workType label

File: src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskProviderTest.java [hunk 3]
```diff
@@ -224,6 +225,7 @@ private Task createTaskWithWarnings() {
             new WarningValues(warnings),
             "Case Management Category",
             "hearing_work",
+            "Hearing work",
```
Comment (BenNzewi) [inline src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskProviderTest.java:228]: could we also update the consumer (TaskManagerGetTaskConsumerTest) to now include the workType label

**Review by BenNzewi — COMMENTED** [review]
Could we update all search 3 search Task FT's relevant methods to include workType label & workType id? I.e.

PostTaskSearchControllerCFTTest
- should_return_a_200_with_search_results_and_correct_properties to include and assert 

GetTaskByIdControllerTest
- should_return_a_200_with_task_and_correct_properties

PostTaskForSearchCompletionControllerCFTTest
- should_return_200_with_appropriate_task_to_complete

**Review by serdaroguzhanakin — DISMISSED** [review]

**Review by dharmendrak — DISMISSED** [review]

**Review by rasikasri — APPROVED** [review]

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**